localstack-core/localstack/services/lambda_/event_source_mapping/esm_event_processor.py

```diff
@@ -21,13 +21,15 @@
 
 class EsmEventProcessor(EventProcessor):
     sender: Sender
+    # TODO: Why have a pipe logger here if pipes is not implemented in community?
```

Comment:
for systematic LS debug logging in the parent `localstack.services.lambda_.event_source_mapping.pipe_loggers.pipe_logger.PipeLogger.log`. That's a bit implicit. any suggestions to clarify are welcomed :)